appveyor.yml

```diff
@@ -5,6 +5,11 @@ environment:
 
 install:
 
+cache:
+  - nsissetup.exe -> appveyor.yaml
+  - NsProcess.zip -> appveyor.yaml
+  - thirdparty\windows -> thirdparty\fetch-thirdparty-deps.ps1

```

Comment:
Ok, this may end up completely breaking anyway so lets try it as-is to start with.